main.go

```diff
@@ -252,6 +254,9 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
+	log.Base().SetLevel(*logLevel)
```

Comment:
You think we should convert the `init` function in `common/log` to use kingpin, rather than remove it? My impression from @Gouthamve was that this was not the intent? I do agree that having this boilerplate is a bit tedious, but from looking at a few different exporters, it looks like most of them are copied and adapted from another one anyway, so it might not be that bad?